app/pb/threads.proto

```diff
@@ -51,9 +59,6 @@ message GetThreadRes {
   // sorted in inverse time order, latest reply first.
   repeated Reply replies = 1;
 
-  // number of responses (comments and replies) associated with the thread
-  uint32 num_responses = 2;
-
```

Comment:
So `TopLevelThread` returns total number of responses for the recursive thread, and `Reply` contains the num_replies for comments?

Reply:
Yes, exactly. There's no way to get the number of comments (without counting replies to comments), but that could be one thing to be added to TopLevelThread in the future.

Reply:
That's not a problem, I think. I don't think we need that (# comments without replies).

The main thing was to make sure we return number of replies for each comment as well.